lib/parse-model.js

```diff
@@ -33,16 +33,33 @@ function parseModel(model) {
 
 
 /**
- * parseArray - Given a model and options, iterates 'size' times to generate
- * a list of the parsed 'model' type.
+ * parseArray - Takes a model, options, and size. If the size is an object (ex. [1, 20])
+ * it will use the randomBetween method from numbers to get a random number between
+ * the first and second index.
+ * If size is a simple number, it just uses that.
  * -> parseModel can be either a Node or a Leaf.
  *
  * @param  {Object} model   A model Node
  * @param  {Object} options [size: Number]
  * @return {Array} A parsed model
  */
 function parseArray(model, options) {
-  return [...Array(options.size).keys()].map(() => parseModel(model.value));
+  let len;
+  let size = options.size;
+  if (Array.isArray(size)) {
+    len = numbers.randomBetween(size)
+  }
+  return [...Array(len).keys()].map(() => parseModel(model.value));
```

Comment:
I've noticed it's not properly working for Array **as a number**. If `len` is not an Array then it will pass `undefined` to `Array(len).keys()...`, and will create an array of length `1`.
I replacing `let size = opti...` to `let len = opti...` could do the trick

Reply:
Yea... I'm an idiot. Rushed to fast and missed that. Will write some tests for it but I did test the edit I just made in my app and it's working now with both size as a Number and size as an Array:

```
{
  "config": {
    "locale": "en"
  },
    "platforms": {
      "type": "Array",
      "options": {
        "size": 2
      },
      "value": {
        "type": "Object",
        "value": {
          "name": {
            "type": "String",
            "value": "internet.domainWord"
          }
        }
      }
    },
    "accounts": {
      "type": "Array",
      "options": {
        "size": [4, 15]
      },
      "value": {
        "type": "Object",
        "value": {
          "name": {
            "type": "faker",
            "value": "internet.userName"
          }
        }
      }
    }
  }
}
```

Reply:
Yeah, I've tested it and works nicely.
That's fine for now in my opinion. Maybe in the future we can enforce unit testing by adding test coverage.